_includes/footer.html

```diff
@@ -13,7 +13,7 @@
   </div>
   <div class="fx-col-100-xs fx-col-33-s fx-col-center-xs">
     <a href="https://github.com/jamulussoftware/jamulus/discussions" target="_blank" rel="noreferrer">
-      <img src="https://icongr.am/fontawesome/question-circle-o.svg?size=35&color=currentColor" alt="{{ site.data.general.footer.alt.sourceforge }}" class="fx-center" id="sflogo" />
+      <img src="https://icongr.am/fontawesome/question-circle-o.svg?size=35&color=currentColor" alt="{{ site.data.general.footer.alt.help }}" class="fx-center" id="sflogo" />
```

Comment:
Is `sflogo` still appropricate?

Reply:
Oh I didn't see that. What does the id here do, I wonder?